inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/PdfAnnoRenderer.java

```diff
@@ -157,11 +158,11 @@ private static Span convertToPdfAnnoSpan(VSpan aVSpan, String aColor, String aDo
             } catch (NoMatchFoundException e) {
                 // if no match is found stop search here. increasing context won't help
                 LOG.error("Could not find a match for existing annotation with id "
-                    + aVSpan.getVid().toString());
+                    + aVSpan.getVid().toString() + " and text \"" + annotatedText + "\"");
                 break;
             }
         }
-        return null;
+        return new Span(aVSpan.getVid().getId(), -1, null, annotatedText, -1, -1);
```

Comment:
Why use only the ID part of the VID and not the entire VID? The ID alone is not guaranteed to be unique. E.g. an annotation might have the same ID as a suggestion.

Reply:
Oh, I wasn't aware of that.
Do I get the entire VID via `toString` method?